File: app/src/main/java/com/divercity/app/features/profile/profileprompt/ProfilePromptActivity.java
package com.divercity.app.features.profile.profileprompt;

import android.content.Context;
import android.content.Intent;

import com.divercity.app.core.base.BaseFragment;
import com.divercity.app.core.base.BaseFragmentActivity;

/**
 * Created by lucas on 17/10/2018.
 */

public class ProfilePromptActivity extends BaseFragmentActivity {

    public static Intent getCallingIntent(Context context){
        return new Intent(context, ProfilePromptActivity.class);
    }

    @Override
    protected BaseFragment fragment() {
        return ProfilePromptFragment.newInstance();
    }
}

File: app/src/main/java/com/divercity/app/features/onboarding/profileprompt/ProfilePromptActivity.kt
package com.divercity.app.features.onboarding.profileprompt

import android.content.Context
import android.content.Intent
import com.divercity.app.core.base.BaseActivity
import com.divercity.app.core.base.BaseFragment

class ProfilePromptActivity : BaseActivity() {

    companion object {

        fun getCallingIntent(context: Context) : Intent {
            return Intent(context, ProfilePromptActivity::class.java)
        }
    }

    override fun fragment(): BaseFragment = ProfilePromptFragment.newInstance()
}
